### PR TITLE
feat(build): record angular version in the dom

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

**Comment by mhevery** [comment]
Please wrap in object, LGTM otherwise.

**Comment by IgorMinar** [comment]
have we considered attaching the version info to the root element(s) instead of body? 

I think it would be useful to  clearly see what is/are the application root(s) and in the future if we allow multiple versions of Angular on the same page (like is being proposed for SharePoint), we'd naturally scope the versions to root elements and in that case we'd have various root elements tagged with their corresponding versions.

File: modules/@angular/core/src/version.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * @stable
+ */
+export class Version {
+  constructor(public full: string) {}
+
+  get major(): string { return this.full.split('.')[0]; }
+
+  get minor(): string { return this.full.split('.')[1]; }
+
+  get patch(): string { return this.full.split('.')[2]; }
```
Comment (gkalpak) [inline modules/@angular/core/src/version.ts]: Is it intended that `patch` will include pre-release identifiers? E.g. for `new Version('2.0.0-rc.1')`, I would expect `patch` to be `'0'`, nit `'0-rc.1'`.

**Comment by born2net** [comment]
so how do we use this new feature?
I'd like to show ng2 version in my help > about web app
regards

**Pull request merged**